parser/src/python.lalrpop

```diff
@@ -787,6 +787,14 @@ MulOp: ast::Operator = {
     "@" => ast::Operator::MatMult,
 };
 
+NamedExpression: ast::Expression = {
+    <e1:Expression> <location:@L> ":=" <e2:Expression> => ast::Expression {
```

Comment:
Please keep in mind here to check with this page: https://docs.python.org/3/reference/grammar.html?highlight=grammar

That's the reference I used when making the rest of this file. Instead of `<e1:Expression>` you might want to go for `<e1:Test>`.